package.json

```diff
@@ -8,7 +8,7 @@
     "build": "yarn build:rollup && yarn build:next",
     "build:next": "cd ./site && next build && next export",
     "build:rollup": "rollup --config ./config/rollup/rollup.config.js",
-    "clean": "rm -rf ./node_modules ./packages/*/{dist,lib,node_modules} ./site/{.next,out}",
+    "clean": "rimraf ./node_modules ./packages/*/{dist,lib,node_modules} ./site/{.next,out}",
     "fix": "yarn fix:prettier && yarn fix:eslint",
```

Comment:
minor change, `rimraf` is a command built into npm/yarn that can do a cross paltform `rm -rf`

Reply:
It's not a built-in, it's just a package that is so common it turns up everywhere. I recommend adding it as an explicit dependency if you're going to use it.
https://www.npmjs.com/package/rimraf

`yarn why rimraf` shows it's currently hoisted from numerous transitive dependencies :)

Reply:
Thanks for catching this mistake! That could have been a little unpleasant 😆